update_existing_families.py

```diff
@@ -104,30 +123,34 @@ def district_school_to_dp_school(name):
     district_record = district_records_dict[student_id]
     if student_id in dp_records_multidict:
         continue
+    donor_ids_encountered = set()
     for sibling_id in students_by_family_ident[district_record['Family_Ident']]:
         if sibling_id in dp_records_multidict:
-            siblingrecord = dp_records_multidict[sibling_id][0]
-            studentrecord = {
-                'DONOR_ID': siblingrecord['DONOR_ID'],
-                'STU_LNAME': district_record['Student Last Name'],
-                'STU_FNAME': district_record['Student First Name'],
-                'STU_NUMBER': student_id,
-                'SCHOOL': district_school_to_dp_school(district_record['School']),
-                'GRADE': district_record['Grade'],
-                'FIRST_NAME': siblingrecord['FIRST_NAME'],
-                'LAST_NAME': siblingrecord['LAST_NAME'],
-                'SP_FNAME': siblingrecord['SP_FNAME'],
-                'SP_LNAME': siblingrecord['SP_LNAME'],
-                'ADDRESS': siblingrecord['ADDRESS'],
-                'ADDRESS2': siblingrecord['ADDRESS2'],
-                'ADDRESS3': siblingrecord['ADDRESS3'],
-                'ADDRESS4': siblingrecord['ADDRESS4'],
-                'CITY': siblingrecord['CITY'],
-                'OTHER_ID': '',
-                'OTHER_DATE': ''
-            }
-            dp_import_studentrecords.append(studentrecord)
-            break
+            for siblingrecord in dp_records_multidict[sibling_id]:
+                if siblingrecord['DONOR_ID'] not in donor_ids_encountered:
```

Comment:
Ah, I guess there was a bug before-- if incoming student had two or more siblings in the district, they would have had multiple records created. I should have seen that.